src/app/recents/recents.component.html

```diff
@@ -0,0 +1,14 @@
+<app-spinner [loading]="loading"></app-spinner>
+<app-navbar></app-navbar>
+
+<div class="recent-title">{{ title }}</div>
+<div class="recent-tabs">
+  <tabset>
+      <tab customClass="recent-table" *ngFor="let type of types" heading="{{ type }}" id="{{ type }}">
```

Comment:
For the heading, you should use the `nodeTypeDisplay` pipe so that it makes the titles pretty. For example, `BugzillaBug`, becomes `Bugzilla Bug`.